pkg/cmd/account/list/list.go

```diff
@@ -18,7 +18,7 @@ func NewCmdList(client apiclient.ClientFactory) *cobra.Command {
 		`),
 		Aliases: []string{"ls"},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			client, err := client.Get(true)
+			client, err := client.Client(true)
```

Comment:
I refactored this to GetSystemClient and GetSpacedClient so there's going to be loads of merge conflicts. I'm happy to resolve them if you merge yours first

Reply:
👌